.docs/content/2.guide/how-to-troubleshoot

```diff
@@ -0,0 +1,83 @@
+# Troubleshooting
+
+Things can go wrong. As a developer, you know this already. But what to do when something goes wrong in ArmoniK ?
+
+This section will give you tips to check and fix problems for those times when it just won’t work and you do not know why.
+___
+## Your deployment failed or is malfunctioning
+
+First, note that if you deployment is local, MongoDB and Redis certificates are only available for 7 days. Past this period, your deployment might encounter issues.
+Our suggestion is to destroy your deployment and redeploy.
+In the 0.1.0 version of the infrastructure, the default validity is set to 8760 hours (one year). You can set the certificate validity parameter before the deployment in the parameters.tfvars file.
+
+To check if your deployment is ok, you can enter the command:
+
+```bash [shell]
+kubectl get po -A
+```
+
+This line allows you to check the status of all the pods. If every one of them is «Running» or «Completed», your deployment is operational.
+___
+## Notes on Terraform
+If your deployment has an issue and you want to stop it immediately instead of waiting for the 10 minutes timeout, we recommend using the following command:
+
+```bash [shell]
+sudo sysctemctl stop k3s.service
+```
```

Comment:
Should not recommend anymore, as with the version on main, since the backend is kubernetes, it will break

Reply:
Ok. I'll just remove the "Notes on Terraform then" as it does not have a purpose anymore. Unless you have a suggestion to stop a deployment safely?

Reply:
Nope, you can remove it